settings/volunteer.setting.php

```diff
@@ -13,4 +13,90 @@
     'description' => 'Fields which are configured to use the slider widget',
     'help_text' => 'Which fields should use the slider widget?',
   ),
+
+  'volunteer_default_profiles_individual' => array(
+    'group_name' => 'CiviVolunteer Configurations',
+    'group' => 'volunteer_defaults',
+    'name' => 'volunteer_default_profiles_individual',
+    'type' => 'String',
+    'default' => '',
```

Comment:
You've changed the behavior here. If the user takes no action, the default individual profile should be `volunteer_sign_up`, as was previously hardcoded into `civicrm_api3_volunteer_util_getsupportingdata`.